docs/docs/Versioning_And_Upgrade/Lua_Toolkit_To_SDK_Plugin_Migration/Plugin_Operations.md

```diff
@@ -0,0 +1,286 @@
+# Plugin Operations
+
+## Summary
+Included below are all plugin operations that have to do with lua migrations. For regular operation information go [here](/References/Plugin_Operations.md).
+
+
+Plugin Operation | **Required** | Decorator | Delphix Engine Operations
+---------------- | -------- | --------- | -------------------------
+[Lua Repository Data Migration](#lua-repository-data-migration) | **No** | `upgrade.repository(lua_version, MigrationType.LUA)` | [Upgrade](/References/Workflows.md#upgrade)
+[Lua Source Config Data Migration](#lua-source-config-data-migration) | **No** | `upgrade.source_config(lua_version, MigrationType.LUA)` | [Upgrade](/References/Workflows.md#upgrade)
+[Lua Linked Source Data Migration](#lua-linked-source-data-migration) | **No** | `upgrade.linked_source(lua_version, MigrationType.LUA)` | [Upgrade](/References/Workflows.md#upgrade)
+[Lua Virtual Source Data Migration](#lua-virtual-source-data-migration) | **No** | `upgrade.virtual_source(lua_version, MigrationType.LUA)` | [Upgrade](/References/Workflows.md#upgrade)
+[Lua Snapshot Data Migration](#snapshot-data-migration) | **No** | `upgrade.snapshot(lua_version, MigrationType.LUA)` | [Upgrade](/References/Workflows.md#upgrade)
+
+
+## Lua Repository Data Migration
+
+A Lua Repository [Data Migration](/References/Glossary.md#data-migration) migrates repository data from an older [schema](/References/Glossary.md#schema) format defined originally from a lua toolkit to an updated schema format.
```

Comment:
nit: python plugin I guess is implicit here, but we can add "python plugin" at the end of the line (here and other places that talk about source config, snapshot etc) - "....an updated schema format defined in the Python plugin."